deployments/warp_routes/PZETH/ethereum-swell-zircuit-config.yaml

```diff
@@ -16,6 +17,18 @@ tokens:
     collateralAddressOrDenom: "0x9cb41CD74D01ae4b4f640EC40f7A60cA1bCF83E7"
     connections:
       - token: ethereum|ethereum|0x0220b1EA1b56ECE8e2b62c8965659f0A621e9ebd
+      - token: ethereum|swell|0x982AbBb04F91acc47aD0CB0A11f29d50c5007934
+    decimals: 18
+    logoURI: /deployments/warp_routes/PZETH/logo.svg
+    name: "Renzo Restaked LST "
```

Comment:
Is the trailing space supposed to be there?

Reply:
yeah its always been in here. it is derived from the xerc20 name